utils/test-utils/index.js

```diff
@@ -1,5 +1,5 @@
 const merge = require('deepmerge');
-const twing = require('../../../src/ec/.storybook/environment'); // eslint-disable-line import/no-unresolved
+const twing = require('@ecl-twig/ec-storybook/.storybook/environment');
```

Comment:
In `php/php_storybook/scripts/render.js` this is `const twing = require('@ecl-twig/ec-storybook/.storybook/environment.js')` please unify.